Ginger/Ginger/SolutionWindows/TreeViewItems/DataSourceTableTreeItem.cs

```diff
@@ -134,7 +135,26 @@ private void DeleteTable(object sender, RoutedEventArgs e)
             }            
         }
 
-        private void ExportToExcel(object sender, RoutedEventArgs e)
+        public override bool DeleteTreeItem(object item, bool deleteWithoutAsking = false, bool refreshTreeAfterDelete = true)
+        {
+            var repoItem = item as RepositoryItemBase;
+            if (repoItem != null)
+            {
+                if (!deleteWithoutAsking)
+                {
+                    if (Reporter.ToUser(eUserMsgKey.DeleteItem, repoItem.GetNameForFileName()) == Amdocs.Ginger.Common.eUserMsgSelection.No)
+                    {
+                        return false;
+                    }
+                }
+            }
+            mTreeView.Tree.DeleteItemAndSelectParent(this);
+            DSDetails.DSTableList.Remove(DSTableDetails);
```

Comment:
Please add this code to delete inside a function and call the same function here as well as DeleteTable